assets/scss/_custom.scss

```diff
@@ -944,65 +970,76 @@ dl {
     font-size: 1.125em;
   }
 
-  .release-inline-heading, .release-inline-value {
-    display: inline-block
+  .release-inline-heading,
+  .release-inline-value {
+    display: inline-block;
   }
 
   .release-inline-value {
     padding-left: 0.25em;
   }
 
   p {
-     margin-top: 1em;
-     margin-bottom: 1em;
+    margin-top: 1em;
+    margin-bottom: 1em;
   }
 }
 
 .release-highlighted-date {
   font-weight: bold;
 }
 
-
 .no-js .mermaid {
   display: none;
 }
 
 div.alert > em.javascript-required {
   display: inline-block;
   min-height: 1.5em;
-  margin: calc(max(4em, ( 8vh + 4em ) / 2)) 0 0.25em 0;
+  margin: calc(max(4em, (8vh + 4em) / 2)) 0 0.25em 0;
 }
 
 // Consistent spacing for yaml code
 .language-yaml > span {
   height: 1.6em;
 }
 
-.content__box
-{
-    margin: 0 0 20px;
-    padding: 20px;
+.content__box {
+  margin: 0 0 20px;
+  padding: 20px;
 }
-.content__box_lined
-{
-    box-shadow: inset 0 0 0 1px #326de6;
+.content__box_lined {
+  box-shadow: inset 0 0 0 1px #326de6;
 }
-.content__box_fill
-{
-    color: #fff;
-    background: #326de6;
+.content__box_fill {
+  color: #fff;
+  background: #326de6;
 }
 
 // Adjust Bing search result page
 #bing-results-container {
-    padding: 1em;
+  padding: 1em;
+}
+
+.bing-result {
+  margin-bottom: 1em;
+}
+
+.bing-result-url {
+  font-size: 14px;
+}
+
+.bing-result-snippet {
+  color: #666666;
+  font-size: 14px;
 }
+
 #bing-pagination-container {
-    padding: 1em;
-    margin-bottom: 1em;
+  padding: 1em;
+  margin-bottom: 1em;
 
-    a.bing-page-anchor {
-        padding: 0.5em;
-        margin: 0.25em;
-    }
+  a.bing-page-anchor {
+    padding: 0.5em;
+    margin: 0.25em;
+  }
 }
```

Comment:
These look like the changes we actually want.

Reply:
Have updated the PR without improved formatting and fixing syntax issues